client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-placeholder-dialog.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+
+const PaymentMethodStripePlaceholderDialog = ( { translate } ) => {
+	const buttons = [
+		{ action: 'cancel', disabled: true, label: translate( 'Cancel' ), onClick: noop },
+	];
+
+	return (
+		<Dialog additionalClassNames="payments__dialog woocommerce" buttons={ buttons } isVisible>
+			<div className="stripe__method-edit-header">{ translate( 'Stripe' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Stripes' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/store-stripe-connect-integ)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).